testData/highlighting/funcCall.go

```diff
@@ -143,8 +143,8 @@ func bar_m() (int, int) {
 	return 0, 0
 }
 
-func <warning>main127</warning>() {
-	foo_m<error>(bar_m())</error>
+func <warning descr="Unused function 'main127'">main127</warning>() {
```

Comment:
I'll try and remove those, I've understood meanwhile why they appear in the tests since I've wrote this :)